src/panels/config/entities/ha-config-entities.ts

```diff
@@ -171,6 +171,7 @@ export class HaConfigEntities extends SubscribeMixin(LitElement) {
         type: "icon",
         template: (_, entry: any) => html`
           <ha-state-icon
+            .title=${entry.entity.state}
```

Comment:
Just thinking about that some more: Won't that be weird if the screen reader suddenly reads the state value "123" instead of making it clear that this is an icon? Or even weirder a whole technical timestamp?
![grafik](https://user-images.githubusercontent.com/114137/150841848-814fe35f-e3fa-43be-8866-6460e1f05d6b.png)


Or will it read "Icon labeled 123"?

Reply:
Screen reader specifies it is an image.
For example in the case of a nest Mini it is pronounced "idle image".